book.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'item'
+require 'securerandom'
+
+class Book < Item
+  attr_accessor :author, :publisher, :cover_state, :publish_date, :label
+  attr_reader :id
+
+  def initialize(_id, author, publisher, cover_state, publish_date, label)
+    super()
+    @id = generate_random_id
+    @author = author
+    @publisher = publisher
+    @cover_state = cover_state
+    @publish_date = publish_date
+    @label = label
+  end
+
+  def generate_random_id
+    SecureRandom.uuid
+  end
+
+  def can_be_archived?
+    if @cover_state || super == 'bad'
+      false
+    else
+      true
+    end
+  end
```

Comment:
:red_circle: [REQUIRED] Kindly make sure you correct this method as suggested by the previous reviewer. 

> This method should return true if parent's method returns true OR if cover_state equals to "bad", otherwise it returns false:
> 
> ```
> def can_be_archived?
>     if @cover_state == bad || super
>       true
>     else
>       false
>     end
>   end
> ```
> 
> If you want to make shorter, this will do the trick:
> 
> ```
>     def can_be_archived?
>       super || @cover_state == 'bad'
>     end
> ```

Reply:
@bienvenuushindi 
Please kindly advice on how we can submit the changes for a third review?
We have lost access to the review option on the dashboard.

Reply:
@RileyManda Yesterday, I mistakenly approved the project while I was still reviewing it. I'm sorry for the inconvenience."I will re-review the project, and once I confirm that all the requested changes have been implemented, you can proceed with the submission.

Reply:
Hi @bienvenuushindi
I have updated book.rb and surrounded bad with quotes 'bad'.
Please kindly review and approve if all is in order.Because the system has already moved our team to the next project.
Thank you